index.d.ts

```diff
@@ -1406,16 +1410,16 @@ declare module 'evergreen-ui' {
   export class SegmentedControl extends React.PureComponent<SegmentedControlProps> {
   }
 
-  export interface SelectProps extends Omit<BoxProps<'div'>, 'onChange'> {
+  export interface SelectProps extends Omit<React.ComponentPropsWithoutRef<typeof Box>, 'onChange'> {
     /**
      * The initial value of an uncontrolled select
      */
-    defaultValue?: string
+    defaultValue?: string | number | string[]
```

Comment:
Could this also be a `number[]`?

Reply:
I grabbed these types from React itself, so Imma trust their types :)